ArduPlane/tailsitter.cpp

```diff
@@ -267,6 +267,14 @@ bool QuadPlane::in_tailsitter_vtol_transition(void) const
     return is_tailsitter() && in_vtol_mode() && transition_state == TRANSITION_ANGLE_WAIT_VTOL;
 }
 
+/*
+  return true if we are a tailsitter in FW flight
+ */
+bool QuadPlane::in_tailsitter_fw_flight(void) const
+{
+    return is_tailsitter() && !in_vtol_mode() && transition_state != TRANSITION_ANGLE_WAIT_FW;
+}
```

Comment:
shouldn't this be == TRANSITION_DONE ?

Reply:
I did wonder about that, my thinking was that we might add more transition states in the future, and this would be another place to have to come and change it. Might be nicer if this was a forward transition check but the `in_tailsitter_fw_flight` was a nicer match to replace the `tailsitter_transition_fw_complete`. I could easy be persuaded to change it.

Reply:
it seems to me that if you are checking for TRANSITION_DONE, you are less likely to need to change this code if new transition states are added. 
That's assuming TRANSITION_DONE actually means not_in_(any current or future)_transition and !in_vtol_mode() means in_fw_flight.

Reply:
I was thinking of a 'accelerating in forward flight' transitioning stage, that would be forward flight but still transitioning.

Anyway, I will change it to `TRANSITION_DONE` as you say, no point speculating about future work.